amundsen_application/static/js/components/SearchPage/styles.scss

```diff
@@ -4,36 +4,23 @@
   display: flex;
   min-height: calc(100% - #{$nav-bar-height + $footer-height});
 
+  .list-group {
+     margin-top: 0px;
```

Comment:
nit: I prefer to use `0` instead of `0px` when possible. Something in my is conditioned to prefer it that way... Not sure why.